acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -63,7 +63,7 @@ test:
     - venv/bin/pip install coverage
     - venv/bin/pip install --no-deps --no-index release/*.whl
     # although the project is part of *.whl, install it in editable mode so coverage sees it
-    - venv/bin/pip install --no-deps --no-index -e .
+    - venv/bin/pip install --no-index -e .
```

Comment:
Add a comment "# use --no-index so missing dependencies that would not be in *.whl are detected". 

Actually the same reasoning should apply to the install of release/*.whl